sdk/resourcemanager/Azure.ResourceManager.Core/src/Operation/PhValueArmOperation.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A calss representing an arm operation wrapper object.
+    /// </summary>
+    internal class PhValueArmOperation<TOperations> : ArmOperation<TOperations>
```

Comment:
Same comment - assuming this is temporary

Reply:
And on modality - would prefer two implementing classes, one for each wrapped entity